docs/ReleaseNotes.md

```diff
@@ -12,12 +12,15 @@ In Acuminator 2.1.3, the following enhancements have been implemented:
  - Now a developer can collapse or expand the selected node (with all its descendant nodes) in the code map if the developer clicks **Collapse All** or **Expand Add** (respectively) in the context menu for the selected node.
  - Multiple changes have been made to the [PX1007](diagnostics/PX1007.md) diagnostic to reduce the number of warnings displayed by the diagnostic. For more information about the situations when the diagnostic is not displayed, see the description of the diagnostic in [PX1007](diagnostics/PX1007.md).
  - If a developer suppresses the [PX1007](diagnostics/PX1007.md) diagnostic with a comment, the default comment message is now the following: _Acuminator disable once PX1007 NoXmlCommentForPublicEntityOrDacProperty to be documented later_.
+ - Improved slightly performance of the [PX1021](diagnostics/PX1021.md)/[PX1023](diagnostics/PX1023.md)/[PX1095](diagnostics/PX1095.md) diagnostics
```

Comment:
It seems that the PR with release notes and Ksenia's comments should be merged first since both PRs contain these changes.